meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -23,12 +24,16 @@ PACKAGECONFIG_append_class-target = " dockerapp"
 PACKAGECONFIG_remove_class-target_riscv64 = "dockerapp"
 
 SYSTEMD_PACKAGES += "${PN}-lite"
-SYSTEMD_SERVICE_${PN}-lite = "aktualizr-lite.service"
+SYSTEMD_SERVICE_${PN}-lite = "aktualizr-lite.service aktualizr-lite.path"
 
 do_install_append() {
     install -d ${D}${systemd_system_unitdir}
     install -m 0644 ${WORKDIR}/aktualizr-lite.service ${D}${systemd_system_unitdir}/
+    install -m 0644 ${WORKDIR}/aktualizr-lite.path ${D}${systemd_system_unitdir}/
 }
 
 # Force same RDEPENDS, packageconfig rdepends common to both
 RDEPENDS_${PN}-lite = "${RDEPENDS_aktualizr}"
+
+FILES_${PN}-lite += "${systemd_system_unitdir}/aktualizr-lite.path"
+FILES_${PN}-lite += "${systemd_unitdir}/system-preset"
```

Comment:
Is this last line really needed here?

Reply:
Yes, I had QA build check errors if for some reason this package gets built before ostree or one of the other packages which has a FILES_${PN} entry similar to this.

Reply:
Specifically there is a file: `/usr/lib/systemd/system-preset/98-aktaulizr-lite.preset` which gets generated with the "enable" lines for these services.

Reply:
Interestingly, most systemd services don't have this.  Not sure why.

Reply:
Gets added automatically by the systemd bbclass:

```
    def systemd_create_presets(pkg, action):
        presetf = oe.path.join(d.getVar("PKGD"), d.getVar("systemd_unitdir"), "system-preset/98-%s.preset" % pkg)
        bb.utils.mkdirhier(os.path.dirname(presetf))
        with open(presetf, 'a') as fd:
            for service in d.getVar('SYSTEMD_SERVICE_%s' % pkg).split():
                fd.write("%s %s\n" % (action,service))
        d.appendVar("FILES_%s" % pkg, ' ' + oe.path.join(d.getVar("systemd_unitdir"), "system-preset/98-%s.preset" % pkg))
```

Reply:
Removed the last 2 FILES_* lines.